src/kernel/src/process/module/recompiler.rs

```diff
@@ -136,28 +136,45 @@ impl<'input> Recompiler<'input> {
             // Transform instruction.
             let (size, end) = match i.code() {
                 Code::Add_r32_rm32 => (self.transform_add_r32_rm32(i), false),
-                Code::Add_rm8_r8 => (self.transform_add_rm8_r8(i), false),
+                Code::Add_r64_rm64 => (self.transform_add_r64_rm64(i), false),
+                Code::Add_rm8_r8 => (self.transform_add_rm8_r8(i), true),
```

Comment:
This one cannot mark and ended. The only kind that can mark as ended is unconditional control transfer without return back.